mock_drs/specs/schema.data_repository_service.cd0186f.openapi.modified.yaml

```diff
@@ -332,7 +371,8 @@ definitions:
           accession numbers or external GUIDs.
   AccessURL:
     type: object
-    required: ['url']
+    # required:
+      # - url
```

Comment:
This required field of `url` is causing validation error, is spite of providing `url`.

This was request body used
```
{
    "id": "a044",
    "name": "README",
    "size": 565,
    "created": "2019-05-20T00:12:34-07:00",
    "updated": "2019-04-24T05:23:43-06:00",
    "version": "1",
    "mime_type": "",
    "checksums": [
      {
        "checksum": "18c2f5517e4ddc02cd57f6c7554b8e88",
        "type": "md5"
      }
    ],
    "access_methods": [
      {
        "type": "ftp",
        "access_url": {
          "url": "ftp://ftp.ensembl.org/pub/release-81/bed/ensembl-compara/11_teleost_fish.gerp_constrained_eleme",
          "headers":  [
            "None"
          ]
        },
        "access_id": "004"
      }
    ]
  }
```
The error coming is: 
```
mock-drs_1  | http://localhost:9101/ga4gh/drs/v1/objects validation error: 'url' is a required property
mock-drs_1  | 
mock-drs_1  | Failed validating 'required' in schema:
mock-drs_1  |     {'properties': {'headers': {'description': 'An optional list of '
mock-drs_1  |                                                'headers to include in the '
mock-drs_1  |                                                'HTTP request to `url`. '
mock-drs_1  |                                                'These headers can be used '
mock-drs_1  |                                                'to provide auth tokens '
mock-drs_1  |                                                'required to fetch the '
mock-drs_1  |                                                'object bytes.',
mock-drs_1  |                                 'example': {'Authorization': 'Basic '
mock-drs_1  |                                                              'Z2E0Z2g6ZHJz'},
mock-drs_1  |                                 'items': {'type': 'string'},
mock-drs_1  |                                 'type': 'array'},
mock-drs_1  |                     'url': {'description': 'A fully resolvable URL that '
mock-drs_1  |                                            'can be used to fetch the '
mock-drs_1  |                                            'actual object bytes.',
mock-drs_1  |                             'type': 'string'}},
mock-drs_1  |      'required': ['url'],
mock-drs_1  |      'type': 'object'}
mock-drs_1  | 
mock-drs_1  | On instance:
mock-drs_1  |     {'access_methods': [{'access_id': '004',
mock-drs_1  |                          'access_url': {'headers': ['None'],
mock-drs_1  |                                         'url': 'ftp://ftp.ensembl.org/pub/release-81/bed/ensembl-compara/11_teleost_fish.gerp_constrained_eleme'},
mock-drs_1  |                          'type': 'ftp'}],
mock-drs_1  |      'checksums': [{'checksum': '18c2f5517e4ddc02cd57f6c7554b8e88',
mock-drs_1  |                     'type': 'md5'}],
mock-drs_1  |      'created': '2019-05-20T00:12:34-07:00',
mock-drs_1  |      'id': 'a044',
mock-drs_1  |      'mime_type': '',
mock-drs_1  |      'name': 'README',
mock-drs_1  |      'size': 565,
mock-drs_1  |      'updated': '2019-04-24T05:23:43-06:00',
mock-drs_1  |      'version': '1'}
mock-drs_1  | 172.20.0.1 - - [20/Jun/2020 10:56:45] "POST /ga4gh/drs/v1/objects HTTP/1.1" 500 -


```